src/main/webapp/popup.js

```diff
@@ -31,7 +32,7 @@ function createNewEvent() {
     }
 
     const event = new Event(title, location, startTime, endTime);
-    addEvent(event);
+    addEventToLocalStorage(JSON.stringify(event));
```

Comment:
Shouldn't JSON.stringify happens under the hood of addEventTolocalstorage?

Reply:
I chose this method because I am using the addEventToLocalStorage() function in other file, where I only use JSON objects. SO the functions receives Json objects

Reply:
Please add a todo and refactor it as follow up in both files.